libexec/rtld-elf/aarch64/reloc.c

```diff
@@ -31,10 +31,12 @@
 #include <sys/cdefs.h>
 __FBSDID("$FreeBSD$");
 
+#include <sys/mman.h>
```

Comment:
sys/param.h and sys/types.h are special for style(9), move this include after them